be/src/exprs/agg/hll_uniq.h

```diff
@@ -0,0 +1,150 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include "column/binary_column.h"
+#include "column/object_column.h"
+#include "column/type_traits.h"
+#include "column/vectorized_fwd.h"
+#include "exprs/agg/aggregate.h"
+#include "gutil/casts.h"
+
+namespace starrocks {
+
+/**
+ * RETURN_TYPE: TYPE_BIGINT
+ * ARGS_TYPE: ALL TYPE
+ * SERIALIZED_TYPE: TYPE_VARCHAR
+ */
+template <LogicalType LT, typename T = RunTimeCppType<LT>>
+class HllUniqAggregateFunction final
```

Comment:
duplicate code with HllNdvAggregateFunction?

Reply:
Yes, 99% of the code are the same right now.

But may be the implementation might be different in the future. Datasketch HLL is a different structure from the current implementation, and the performance is not good for now. Some codes need to be changed later.